fairscale/nn/wrap/auto_wrap.py

```diff
@@ -229,6 +229,7 @@ def recursive_wrap(module: nn.Module, auto_wrap_policy: Optional[Callable], **kw
             auto_wrap_policy = ConfigAutoWrap.auto_wrap_policy
 
         # Make sure no child is not already wrapped.
+        # TODO (Min): bug #649
```

Comment:
I probably should remove this. not much info but I was marking it so that I don't forget. :-)